CHANGELOG.md

```diff
@@ -3,12 +3,22 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
+
+## [2.2.1]
 ### Changed
-* Updated dependencies
+* Updated dependencies ([#246])
+* Erroring test reading symbolic link to dir ([#229])
```

Comment:
Ok, sorry I only read the code and the summary for that PR. We need a better process for this.

1. un-publish `2.2.1` from npm
2. correct changelog entry version
3. bump to `2.3.0`.
4. republish.

Agree? Did I miss anything?